dependencyManagement/build.gradle.kts

```diff
@@ -58,6 +58,7 @@ val DEPENDENCIES = listOf(
   "javax.annotation:javax.annotation-api:1.3.2",
   "com.github.stefanbirkner:system-rules:1.19.0",
   "com.google.api.grpc:proto-google-common-protos:2.49.0",
+  "com.google.code.findbugs:annotations:3.0.1",
```

Comment:
Seems this new version of equalsverifier requires an additional dependency. Opened an issue to clarify whether this is expected: https://github.com/jqno/equalsverifier/issues/1026